src/core/hle/service/nwm/nwm_uds.cpp

```diff
@@ -928,6 +931,57 @@ void NWM_UDS::BeginHostingNetworkDeprecated(Kernel::HLERequestContext& ctx) {
     rb.Push(result);
 }
 
+void NWM_UDS::EjectClient(Kernel::HLERequestContext& ctx) {
+    IPC::RequestParser rp(ctx, 0x05, 1, 0);
+    u16 network_node_id = rp.Pop<u16>();
+
+    LOG_WARNING(Service_NWM, "(stubbed) called");
+
+    IPC::RequestBuilder rb = rp.MakeBuilder(1, 0);
+
+    // The host can not be kicked.
+    if (network_node_id == 1) {
+        rb.Push(ResultCode(ErrorDescription::NotAuthorized, ErrorModule::UDS,
+                           ErrorSummary::WrongArgument, ErrorLevel::Usage));
+        return;
+    }
+
+    std::lock_guard<std::mutex> lock(connection_status_mutex);
+    if (connection_status.status != static_cast<u8>(NetworkStatus::ConnectedAsHost)) {
+        // Only the host can kick people.
+        rb.Push(ResultCode(ErrorDescription::NotAuthorized, ErrorModule::UDS,
+                           ErrorSummary::InvalidState, ErrorLevel::Usage));
+        LOG_WARNING(Service_NWM, "called with status {}", connection_status.status);
+        return;
+    }
+
+    // This function always returns success if the status is valid.
+    rb.Push(RESULT_SUCCESS);
+
+    using Network::WifiPacket;
+    Network::MacAddress dest_address{};
```

Comment:
`{}` doesn't do anything
```suggestion
    Network::MacAddress dest_address;
```